GeoBot/GeoBot/Startup.cs

```diff
@@ -32,6 +32,16 @@ public Startup(IConfiguration config)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+
+            services.AddCors(options =>
```

Comment:
Idea:
Would like to integrate the WebChat html/javascript file into the webroot of the bot, that should be enough to remove the CORS allowance for all possible websites. CORS related part could be commented out with a hint that if you would like to call it from another webpage CORS allowance for that domain/all has to be activated again.

Background:
CORS does not bring really more security since it can be bypassed by raw HTTP calls, but it reduces the possible attack surface within standard browsers. So I would like to give out a version where CORS allow all is disabled by default.

Reply:
I totally agree on your suggestion! I will enable this while testing and disable this option at the final version of code.